src/app/app.component.css

```diff
@@ -24,7 +24,7 @@
     background-color: #FFC8C8;
 }
 
-.styling {
+.stylistic {
```

Comment:
This CSS class doesn't seem to be used anywhere inside this repo. Then do we really need it ?

In changing the name of a CSS class, I would expect at least two changes, one where its defined and one where its used. Why is it not the case here, and how is a FE CSS class related to a constant variable in the backend ? ℹ️

Reply:
This class is used, e.g. [here](https://github.com/OpenCovenant/quill-fe/blob/main/src/app/app.component.ts#L99) and [here](https://github.com/OpenCovenant/quill-fe/blob/main/src/app/app.component.ts#L192). So, the type of the marking dictates that CSS class. I assume another way would be to have some sort of mapping from the marking to the class to be used, which would be resolved in the FE. If you want we can create a PR and do the latter. Either is fine with me, no strong preference for now.